theme/global/full-page-styles.ts

```diff
@@ -1,6 +1,6 @@
 export const fullPageStyles = {
   '.mode__full-page': {
-    '&, body, #app, main,.radix-themes': {
+    '&, body, main, .radix-themes': {
```

Comment:
cc @pete-watters I also had to make this change, or I saw this on my branch...

<img width="756" alt="Screen Shot 2023-09-14 at 3 23 55 PM" src="https://github.com/leather-wallet/extension/assets/6493321/c31cb293-3982-4ea2-9cde-b0c5cfe2b391">

Reply:
Thanks! Again sorry for this introduced regression. A few backward steps to try jump forward.

I'm not sure what happened here. Maybe that `#app,` syntax was disabling something else?